src/components/UserInputComponent/WayareaComponent.js

```diff
@@ -1,18 +1,43 @@
-import { Autocomplete, TextField, Typography, Tooltip } from "@mui/material";
+import {
+  Autocomplete,
+  TextField,
+  Typography,
+  Tooltip,
+  Grid,
+  InputLabel,
+  Select,
+  MenuItem,
+  IconButton,
+} from "@mui/material";
+import ClickAwayListener from "@mui/material/ClickAwayListener";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
 import { useContext, useEffect, useState } from "react";
 import apiClient from "http-common";
 import NotificationContext from "contexts/NotificationContext";
 import { useField } from "formik";
 import Form from "react-bootstrap/Form";
 import SelectedWayareaContext from "contexts/SelectedWayareaContext";
 
-export default function WayareaNameComponent(props) {
-  const { name } = props;
+export default function WayareaComponent(props) {
+  const { name, formik } = props;
   const [field, meta] = useField(name);
   const [allWayareas, setAllWayareas] = useState([]);
+  const [open, setOpen] = useState(false);
   const { setNotificationStatus } = useContext(NotificationContext);
   const { setSelectedWayarea } = useContext(SelectedWayareaContext);
   const [wayareaInputString, setWayareaInputString] = useState("");
+  const [calculationInterval, SetCalculationInterval] = useState(10);
+  const calculationIntervalOptions = [
+    10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 250, 500, 750, 1000,
+  ];
+
+  function setChosenWayareaFormikValue(wayarea) {
+    if (wayarea) {
+      formik.setFieldValue("navline.VAYLAT", wayarea.VAYLAT);
+    } else {
+      formik.setFieldValue("navline.VAYLAT", "");
+    }
+  }
```

Comment:
```suggestion
  function setChosenWayareaFormikValue(wayarea) {
      formik.setFieldValue("navline.VAYLAT", wayarea.VAYLAT);
  }
```
This function could be changed to always set the given parameter to formik.

Reply:
still needs logic to check for undef values